python/convert-from-gadget-1.py

```diff
@@ -15,6 +17,19 @@ def main(ns):
     cat = Gadget1Catalog(ns.source, ptype=1)
 
     attrs = cat.attrs.copy()
+    
+    new_cat = cat.attrs.copy()
+    cat.attrs.clear()
+    
+    cat.attrs['a.x'] = new_cat['Time']
+    cat.attrs['a.v'] = new_cat['Time']
+    cat.attrs['M0'] = new_cat['Massarr'][1]
+    cat.save(ns.dest, columns=[], dataset='1', header='Header')
+    odir = ns.dest+'/1'
```

Comment:
I think you can avoid shutil (and fiddling the internals of bigfile) by doing this:

```
import bigfile
```

```
with bigfile.File(ns.dest) as ff:
   with ff['1'] as column:
     column.attrs['.v'] = cat['Time']
     ....
```